plugins/koa/koaintegrationplugin.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+const compose = require('koa-compose');
+const Router = require('koa-router');
+const router = new Router();
+const reconfig = require('@brillout/reconfig');
+
+module.exports = {
+    name: 'koaIntegrationPluginFile',
+    multiple: false,
+    routes: routes(),
+};
+
+function routes() {
+	return compose(
+	[
+		router.routes(),
+		router.allowedMethods()
+	]
+	)
+}
+
+router.get('*', async(ctx, next) => {
```

Comment:
I don't know koa but shouldn't `next` be called somewhere?

Reply:
No, it's part of the router